airbyte-webapp/src/packages/cloud/services/thirdParty/intercom/useIntercom.ts

```diff
@@ -29,11 +29,11 @@ export const useIntercom = (): IntercomContextValues => {
   useEffect(() => {
     intercomContextValues.update({
       customAttributes: {
-        workspace_id: analyticsContext.workspace_id,
```

Comment:
:information_source: This likely never worked, since there was no `workspace_id` on that analytics context, but since it was also not typed it was never caught. Switched this to use the actual correct hook.